test/functionalTest/cases/2763_postUpdateContext_error_return/invalid_service_path_gives_400_bad_request.test

```diff
@@ -0,0 +1,56 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Looking again, I don't see the .test covering #1985.... Maybe you missed `git add` before commit? ;)